lib/registries/addon/drafts/draft/template.hbs

```diff
@@ -39,31 +39,34 @@
                             @draftManager={{draftManager}}
                             @navManager={{navManager}}
                             @onSubmitRedirect={{this.onSubmitRedirect}}
+                            @readOnly={{draftManager.currentUserIsReadOnly}}
```

Comment:
Nitpick: parameter name could be changed to `currentUserIsReadOnly`. But doesn't really matter that much.